include/micm/process/process_set.hpp

```diff
@@ -218,6 +219,7 @@ namespace micm
         prod_id += number_of_products_[i_rxn];
         yield += number_of_products_[i_rxn];
       }
+      delete[] rate;
```

Comment:
On second thought, please just make rate a vector<double> and reserve `L` spaces. There's no reason to manage the memory ourselves.

Reply:
Sounds good. I updated `rate` to a vector